src/utilities.h

```diff
@@ -662,4 +675,18 @@ ConnectionMap createConnectionMap(const VariablePtr &variable1, const VariablePt
  */
 std::vector<VariablePtr> equivalentVariables(const VariablePtr &variable);
 
+/**
+ * @brief Test the given entities are equal to entities in owner.
+ *
+ * Test to see if all the entites given in @p entities are equal to
+ * entities in @p owner.  The order that the entities appear in is not
+ * taken into account.
+ *
+ * @param owner The owner to compare entities with.
+ * @param entities The list of entities to equate.
+ *
+ * @return @c true if all the entities in @p entities are equal to entites in the @p owner.
+ */
```

Comment:
Is the order important? Or just that the two sets are the same?

Reply:
The latter.

Reply:
Then it should be spelled out in the docs.

Reply:
The documentation of the method says that the order is not important.